.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
```

Comment:
let's use 14 which is LTS now

Reply:
14 is not available for use in github actions at the moment

Reply:
who told you that? :D

Reply:
I got an error earlier when I tried to change it to node 14. Anyways I will check the docs

Reply:
Strange! Docs mention  node-version: [10.x, 12.x, 14.x, 15.x] 
I will change it

Reply:
I got this error after changing to node14 and running the action
`Error: System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter ''using: node14' is not supported, use 'docker' or 'node12' instead.')`

Reply:
https://github.com/fastify/dependabot-merge-action-app/blob/master/.github/workflows/ci.yml#L10